client/src/pages/_app.tsx

```diff
@@ -54,15 +53,7 @@ const retryLink = new RetryLink();
 
 const client = new ApolloClient({
   link: from([retryLink, errorLink, httpLink]),
-  cache: new InMemoryCache({
-    typePolicies: {
-      Query: {
-        fields: {
-          paginatedEvents: offsetLimitPagination(),
-        },
-      },
-    },
-  }),
```

Comment:
needed to delete this, because the data doesn't exist? 

in policies.d.ts

```ts
export declare type FieldReadFunction<TExisting = any, TReadResult = TExisting, TOptions extends FieldFunctionOptions = FieldFunctionOptions> = (existing: SafeReadonly<TExisting> | undefined, options: TOptions) => TReadResult | undefined;
```